index.html

```diff
@@ -44,6 +44,187 @@ <h3 class="headline-intro">
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button class="card-2-button" type="button">See Project</button>
+        <div class="works-card-2">
+          <h5>Profesional Art<br>
+            Printing Data
+          </h5>
+          <div class="card-content-2">
+            A daily selection of privately<br>
+            personalized reads; no accounts or<br>
+            sign-ups required. has been the<br>
+            industry's standard
+          </div>
+          <ul class="tech-stack-2">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button class="card-2-button" type="button">See Project</button>
+        <div class="works-card-2">
+          <h5>Profesional Art<br>
+            Printing Data
+          </h5>
+          <div class="card-content-2">
+            A daily selection of privately<br>
+            personalized reads; no accounts or<br>
+            sign-ups required. has been the<br>
+            industry's standard
+          </div>
+          <ul class="tech-stack-2">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button class="card-2-button" type="button">See Project</button>
+        <div class="works-card-2">
+          <h5>Profesional Art<br>
+            Printing Data
+          </h5>
+          <div class="card-content-2">
+            A daily selection of privately<br>
+            personalized reads; no accounts or<br>
+            sign-ups required. has been the<br>
+            industry's standard
+          </div>
+          <ul class="tech-stack-2">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button class="card-2-button" type="button">See Project</button>
+        <div class="works-card-2">
+          <h5>Profesional Art<br>
+            Printing Data
+          </h5>
+          <div class="card-content-2">
+            A daily selection of privately<br>
+            personalized reads; no accounts or<br>
+            sign-ups required. has been the<br>
+            industry's standard
+          </div>
+          <ul class="tech-stack-2">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button class="card-2-button" type="button">See Project</button>
+      </div>
+    </div>
+  <!--Recent Works Section Ends-->
+  <!--About Me Section Begins-->
+    <div class="about-me-container">
+      <div class="about">
+        <h6>About Me</h6>
+        <div class="about-content">
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </div>
```

Comment:
- [ ] This should be a paragaph. In that case, please use a `p` tag here instead of a `div`